setup.py

```diff
@@ -0,0 +1,60 @@
+#! /usr/bin/env python
+
+import distutils.cmd
+import distutils.log
+import setuptools
+import subprocess
+import os
+
+
+class GrpcCommand(distutils.cmd.Command):
+    """A custom command to generate grpc modules"""
+    user_options = []
+    description = 'download proto file run grpctools'
+
+    def initialize_options(self):
+        self.clean('proto')
+    def clean(self,dir):
+        if not os.path.exists(dir):
+            os.mkdir(dir, 0777)
+        else:
+            for file in os.listdir(dir):
+                file_path = os.path.join(dir, file)
+                try:
+                    if os.path.isfile(file_path):
+                        os.unlink(file_path)
+                except Exception as e:
+                    print(e)
+
+    def finalize_options(self):
+        pass
+
+    def run(self):
+        """Run command."""
+
+        command = ["wget", "https://raw.githubusercontent.com/projectriff/function-proto/master/function.proto", "-P",
```

Comment:
I'm not sure when `GrpcCommand` is used, but there is the same problem @jchesterpivotal pointed out before about this being tied to `master`.

Reply:
This is for ./setup.py grpc invoked manually